libgoods/cli.py

```diff
@@ -144,6 +145,19 @@ def _show_status(main_cat, model_name):
         print(f"{model_name:<20} {timing:<32} {str(status):<6} {str(start):<20} {end}")
 
 
+def _rotate_bbox(model_name, bbox) -> Tuple[float, float, float, float]:
```

Comment:
can this logic be moved to model_fetch?

I'm not sure how the CLI will be used, and the metadata about what models use what system should be retrieved from the catalog, yes?